samples/subsys/display/lvgl/src/main.c

```diff
@@ -15,15 +15,66 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(app);
 
+#if DT_NODE_HAS_STATUS(DT_INST(0, gooddisplay_ek79652), okay)
+#define LVGL_DISPLAY_DEV_NAME DT_LABEL(DT_INST(0, gooddisplay_ek79652))
```

Comment:
No, you can not do it that way. We probably need a chosen property like "zephyr,display-controller".

Reply:
Okay , will make changes based on chosen property in the latter commit